### DISCUSSION TITLE
What to do with OSP_Qualification_Plan_Schema.json?

### DISCUSSION THREAD

**Discussion by PavelBal** [Open-Systems-Pharmacology/QualificationPlan#189] [Q&A]
What do I need it for and where do I put it?

**Comment by PavelBal** [comment]
@Yuri05 tagging you here as I am not sure if you get notifications for discussions.

**Comment by Yuri05** [comment]
Qualification schema is used for validity checks and provides intellisense when **editing** a qualification plan with dedicated tools (e.g. in Visual Studio Code). 
It is **not** used during **processing** of a qualification plan (thus neither _Qualification Runner_ nor _Reporting Engine_ are influenced)

**Reply by Yuri05** [reply]
s. also https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/qualification#creating-a-re-qualification-plan-part-ii-tools

<details><summary><b>Example validation</b></summary>

![grafik](https://user-images.githubusercontent.com/25061876/155692123-1a3c804f-abb6-49f3-a507-f3f7a985f460.png)
</details>
<details><summary><b>Example Intellisense</b></summary>

![grafik](https://user-images.githubusercontent.com/25061876/155691985-3388fb37-d079-4038-a62c-37f24622ed64.png)
</details>

**Comment by PavelBal** [comment]
There is some info about the snippets, what not about the scheme file (searching for the word "scheme" does not return any result). So I am still not sure how to link it to e.g. VSC.

**Reply by Yuri05** [reply]
![grafik](https://user-images.githubusercontent.com/25061876/155698565-ada7f1a5-b6bb-4e9e-a440-d0bf6f498c0a.png)

s. also existing qualification plans, e.g.
https://github.com/Open-Systems-Pharmacology/Qualification-DDI-UGT/blob/f5b1b15d2b99ed7e4763424efa4ce1e96ac42554/Qualification/Input/qualification_plan.json#L2

**Comment by PavelBal** [comment]
Aaaah so it will be generated by the snippet, automatically referencing the latest version! ok.
